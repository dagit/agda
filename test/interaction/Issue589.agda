module Issue589 where

data N : Set where
  zero : N
  suc : N -> N

_+_ : N -> N -> N
zero + y = y
suc x + y = suc ?
