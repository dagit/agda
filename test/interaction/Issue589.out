(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-info-action "*Type-checking*" "Checking Issue589 (Issue589.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue589.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : N\n" nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-give-action 0 "(x + y)")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
